composer.json

```diff
@@ -25,7 +25,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "5.4-dev"
+            "dev-5.x": "5.4-dev"
```

Comment:
@stof Because 5.x is not the same as 5.4.x. Also, all other branches in this repo, and in symfony/symfony follow the convention of aliasing the minor version too.